tests/unit/exporters/test_mlflow_data_exporter.py

```
@@ -637,3 +637,95 @@ def _silent_crash_subprocess_entry(exporter_config: Any, result_queue: Any) -> N
+
+class TestSweepChildNaming:
+    """Tests for _derive_sweep_child_name — names per-probe runs by swept value."""
+
+    def test_concurrency_sweep_names_child_by_concurrency(self, tmp_path: Path) -> None:
+        """Concurrency-based phase produces 'Concurrency=N' name."""
+        cfg = _make_mlflow_cfg(
+            tmp_path,
+            profiling={"type": "concurrency", "requests": 100, "concurrency": 64},
+        )
+        _ = self._make_exporter(cfg, tmp_path)
+        assert exporter._derive_sweep_child_name() == "Concurrency=64"
```

Ruff (0.16.0)

[error] Undefined name `exporter`

(F821)